content/en/tracing/advanced/connect_logs_and_traces.md

```diff
@@ -119,6 +119,26 @@ This enables automatic trace ID injection for `winston`, `bunyan`, and `pino`.
 
 **Note**: Automatic injection only works for logs formatted as JSON.
 
+{{% /tab %}}
+{{% tab ".NET" %}}
+
+Enable injection in the .NET Tracer’s [configuration][1] by setting `DD_LOGS_INJECTION=true` through environment variables or configuration files, or in code:
```

Comment:
Currently, it is only turned on at startup of the tracer. So for now, this only works by setting `DD_LOGS_INJECTION=true.` We can create a backlog item to dynamically add it later in execution by setting the `Tracer.Instance.Settings.LogsInjectionEnabled = true;` but that does not work right now.

Reply:
Oh, right. We talked about this. 🤦‍♂

Reply:
I changed the code sample and added text to clarify that later changes are ignored.